modules/c/examples/code_snippets/main.c

```diff
@@ -1018,7 +1100,44 @@ static void enable_basic_auth() {
     CBLAuth_Free(basicAuth);
 
     CBLReplicator_Start(replicator, false);
-    // end::basic-authentication[]
+    // end::basic-authentication-full[]
+}
+
+static void docsonly_N1QL_Params() {
+    CBLDatabase* database = kDatabase;
+
+    // tag::query-syntax-n1ql-params[]
+    int errorPos;
+
+    CBLError err;
+
+    FLString n1qlstr = FLSTR("SELECT * FROM _ WHERE type = $type");
+
+    FLMutableDict n1qlparams = FLMutableDict_New();
```

Comment:
This is never released.  I want to make sure that things like this are released in our doc snippets so that users don't forget to do it.

Reply:
Ok. I have done that sporadically. I'll ad the releases in.